### PR TITLE
feature: added live server and watch:web command

### PR DESCRIPTION
**Pull request by 32teeth**
## Problem

> [!NOTE] 
> Local dev pain point

The current build setup does not efficiently handle watching files for changes across different parts of the project, such as the main codebase and the demo example. This results in a fragmented development workflow where developers need to manually start and monitor multiple watch tasks.

## Solution

The PR introduces the following changes to streamline the development workflow:

- Added a `watch:demo` script to navigate to the `example` directory and run its watch script.
- Combined the `watch`, `watch:demo`, and `watch:serve` scripts into a single `watch:web` script using `run-p` for parallel execution.
- Added a `watch:serve` script to start a live server on port 9000 serving the `example/dist` directory.

These changes enable developers to start all necessary watch tasks with a single command, ensuring a more efficient and cohesive development experience.

```json
{
    "watch:demo": "cd ./example && npm run watch",
    "watch:web": "run-p watch watch:demo watch:serve",
    "watch:serve": "live-server --port=9000 example/dist"
}
```

> [!TIP] 
> Just run `npm run watch:web` and everyone is happy

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,11 +27,11 @@
     "postinstall": "node postinstall.js"
   },
   "dependencies": {
+    "escape-html": "^1.0.3",
     "just-clone": "^6.2.0",
     "marked": "^12.0.2",
     "prismjs": "1.29.0",
     "sanitize-html": "^2.12.1",
-    "escape-html": "^1.0.3",
```
Comment (32teeth) [inline package.json:31]: Its not missing, its just been alphabetically sorted (see diff)

File: package.json [hunk 2]
```diff
@@ -16,6 +16,9 @@
     "build": "rm -rf dist && webpack --config webpack.config.js --mode production",
     "packdemo": "cd ./example && npm run pack",
     "watch": "webpack --config webpack.config.js --mode development --watch",
+    "watch:demo": "cd ./example && npm run watch",
+    "watch:web": "run-p watch watch:demo watch:serve",
+    "watch:serve": "live-server --port=9000 example/dist",
```
Comment (volodkevych) [inline package.json:21]: nit: some naming considerations:
I'd prefer "watch:demo" renamed to "watch:example" for terminology consistency in the repo. And "watch:serve" renamed to "serve:example".

**Comment by volodkevych** [comment]
Would it make sense to also update example README [here](https://github.com/aws/mynah-ui/tree/main/example)?

File: package.json (hunk 2 above)
Comment (32teeth) [reply package.json:21]: This can be done.
Will push a commit this morning for review

**Review by volodkevych — DISMISSED** [review]

**Review by volodkevych — APPROVED** [review]

**Pull request merged**